src/helpers/types.ts

```diff
@@ -14,6 +14,12 @@ export enum TransactionType {
   CASH_ATM = "CASH_ATM",
   CASH_MANUAL = "CASH_MANUAL",
   CREDIT_PRESENTMENT = "CREDIT_PRESENTMENT",
+  CASH_ATM_REVERSAL = "CASH_ATM_REVERSAL", // ATM-Withdrawal Reversal
```

Comment:
I don't think we need to have our additional description here. Because it's not coming from Solaris directly